src/buildblock/ProjDataInMemory.cxx

```diff
@@ -160,10 +162,9 @@ write_to_file(const string& output_filename) const
 float 
 ProjDataInMemory::get_bin_value(Bin& bin)
 {
-   Viewgram<float> viewgram = get_viewgram(bin.view_num(),bin.segment_num()); 
-    
-   return viewgram[bin.axial_pos_num()][bin.tangential_pos_num()]; 
-
+//   Viewgram<float> viewgram = get_viewgram(bin.view_num(),bin.segment_num());
+//   return viewgram[bin.axial_pos_num()][bin.tangential_pos_num()];
+    return ProjDataFromStream::get_bin_value(bin);
```

Comment:
so, no need for this overload at all now.

Reply:
👍